Adamant/Modules/Chat/View/Subviews/BadgeView.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  BadgeView.swift
+//  Adamant
+//
+//  Created by Аркадий Торвальдс on 26.08.2024.
+//  Copyright © 2024 Adamant. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SnapKit
+
+class BadgeView: UIView {
+    private lazy var counterLabel: UILabel = {
+        let rv = UILabel()
+        rv.numberOfLines = 0
+        rv.lineBreakMode = .byWordWrapping
+        rv.font = .systemFont(ofSize: 12)
+        rv.textColor = labelColor
+        rv.text = String.empty
+        rv.translatesAutoresizingMaskIntoConstraints = false
+        return rv
+    }()
+    
+    private var labelColor: UIColor {
+        UIColor { traits -> UIColor in
+            return traits.userInterfaceStyle == .dark ? UIColor.adamant.first : UIColor.adamant.fourth
```

Comment:
We have already implemented the dark/light theme switch in `UIColor.adamant`

Reply:
Yes, I know that, but among the colors that depend on the theme, I couldn't find a suitable one (in my opinion). The colors .first and .fourth do not change depending on the theme, so I created a private color inside the view.

Reply:
Are you sure that none of the listed colors fit? For example, `.textColor` and `.primary` are already accounted for in the dark/light theme.

Reply:
https://docs.google.com/document/d/16rWyghqJEwwAqbt9sbV0AAFfy4p3koJ0R8YVIbqOPRs/edit#heading=h.wrdiovalu64e
Here you can see all the colors we're using

Reply:
colors changed to .systemRed and .white, as in default badgeValue